@ncksllvn are these test failures related to recent downtime changes?

```
  2) <DowntimeMessage> should render with generic message:
     AssertionError: expected 'This form is down for maintenance.' to include 'We’re sorry it’s not working right now.'
      at Context.<anonymous> (test/common/schemaform/save-in-progress/DowntimeMessage.unit.spec.jsx:14:52)

  3) <DowntimeMessage> should render with window message:
     AssertionError: expected 'This form is down for maintenance.' to include 'We’re sorry it’s not working right now, and we hope to be finished by May 29th, 4:15 p.m.'
      at Context.<anonymous> (test/common/schemaform/save-in-progress/DowntimeMessage.unit.spec.jsx:23:52)
```

--- a/src/applications/personalization/dashboard/tests/containers/DashboardApp.unit.spec.jsx
+++ b/src/applications/personalization/dashboard/tests/containers/DashboardApp.unit.spec.jsx
@@ -6,7 +6,9 @@ import { DashboardApp } from '../../containers/DashboardApp';
 
 
 const props = {
-  profile: {}
+  profile: {
+    loa: {}
+  }
 };
 
 describe('<DashboardApp>', () => {

